tonic/src/lib.rs

```diff
@@ -24,8 +24,11 @@
 //! - `router`: Enables the [`axum`] based service router. Enabled by default.
 //! - `codegen`: Enables all the required exports and optional dependencies required
 //!   for [`tonic-build`]. Enabled by default.
-//! - `tls`: Enables the [`rustls`] based TLS options for the `transport` feature. Not
-//!   enabled by default.
+//! - `tls`: Deprecated. An alias to `tls-ring`
+//! - `tls-ring`: Enables the [`rustls`] based TLS options for the `transport` feature using
+//!   the [`ring`] libcrypto provider. Not enabled by default.
```

Comment:
It's not actually super clear to me _why_ this is failing... perhaps because the `ring` crate isn't a direct dependency? Although then we might expect it to also show a failure for `aws-lc-rs` since that's not a direct dependency either... 🤔

Reply:
I believe this is because `aws-lc-rs` is pulled in by `rustls` as a direct dependency. I added a link to `ring` in the docs to resolve this issue